docs/en_US/bridges/tcp-bridge.md

```diff
@@ -125,6 +125,8 @@ $ nanomq start --old_conf nanomq.conf
 
 ::::
 
+Tips: If you enabled SQLite feature, NanoMQ will automatically flush cached messages into disk when network is disconnected. NanoMQ will resend cached messages once bridging connection is restored. But each cached message will be resent in a certain interval to avoid bandwidth exhaustion.
```

Comment:
shall we explain how to enable sqlite? maybe a link to the ../config-description/broker.md ?